pkg/proxy/winkernel/hns_test.go

```diff
@@ -42,95 +42,65 @@ const (
 	protocol          = 6
```

Comment:
This is mainly tests for hcnutils.go, or? Should we rename this  file to hcnutils_test.go?

Reply:
Yes, you are right. We can name it that way, because that is what it is mainly testing.

Reply:
Yes, you are right. We can name it that way, because that is what it is mainly testing.